streamer/src/nonblocking/quic.rs

```diff
@@ -495,6 +495,16 @@ async fn setup_connection(
                         stats.clone(),
                     ),
                     |(pubkey, stake, total_stake, max_stake, min_stake)| {
+                        // The heuristic is that the stake should be large engouh to have 1 stream pass throuh within one throttle
+                        // interval during which we allow max (MAX_STREAMS_PER_100MS * 100) streams.
+                        let min_stake_ratio = 1_f64 / (MAX_STREAMS_PER_100MS * 100) as f64;
```

Comment:
We should not multiply with 100 here. The constant `MAX_STREAMS_PER_100MS` already accounts for that.

Reply:
Oh right. Silly me!